kernels/portable/cpu/op_grid_sampler_2d.cpp

```diff
@@ -0,0 +1,257 @@
+//
+//  Copyright (c) 2025 Apple Inc. All rights reserved.
+//  Provided subject to the LICENSE file in the top level directory.
+//
+
+#include <executorch/kernels/portable/cpu/scalar_utils.h>
+#include <executorch/kernels/portable/cpu/util/elementwise_util.h>
+#include <executorch/runtime/kernel/kernel_includes.h>
+
+#include <algorithm>
+#include <cmath>
+#include <cstdint>
+#include <utility>
+
+#include <c10/util/irange.h>
+
+namespace torch {
+namespace executor {
+namespace native {
```

Comment:
nit: we have C++17, new code should use nested namespaces (`namespace torch::executor::native {`)